package.json

```diff
@@ -38,8 +38,8 @@
 		"gettext-parser": "^4.0.3",
 		"husky": "^3.0.2",
 		"identity-obj-proxy": "^3.0.0",
-		"jest": "^24.9.0",
-		"jest-junit": "^6.3.0",
+		"jest": "^27.4.5",
+		"jest-junit": "^13.0.0",
```

Comment:
These versions match the values from Gutenberg's `package.json` file:
https://github.com/WordPress/gutenberg/pull/33287/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519